core/src/main/java/com/klaytn/caver/contract/Contract.java

```diff
@@ -379,7 +379,7 @@ private void init(String abi) throws IOException {
                     });
 
                     if(existedMethod.getInputs().size() == newMethod.getInputs().size() || isWarning) {
-                        LOGGER.warn("An overloaded function with the same number of parameters may not be executed normally.");
+                        LOGGER.warn("An overloaded function with the same number of parameters may not be executed normally. Please use *withSolidityWrapper methods in ContractMethod class.");
```

Comment:
How about `findMatchedInstance` function will return `List<ContractMethod>` and if there is more than one ContractMethod is returned from `findMatchedInstance`, throw exception or logger.warn in `send` or `call`?